web/firebase.js

```diff
@@ -58,37 +78,66 @@ export const userSignIn = (email, password)=>{
       const errorCode = error.code;
       const errorMessage = error.message;
     });
-}
+};
 
 //Sign up for a new user
-export const userSignUp = (email, password)=>{
+export const userSignUp = (email, password) => {
   createUserWithEmailAndPassword(auth, email, password)
     .then((userCredential) => {
       const user = userCredential.user;
-      console.log('Created new account for:', user.email);
+      console.log("Created new account for:", user.email);
       userIsAuthenticated = true;
     })
     .catch((error) => {
       const errorCode = error.code;
       const errorMessage = error.message;
     });
-}
+};
 
 //Sign out of user account
-export const userSignOut = ()=>{
+export const userSignOut = () => {
   const authInfo = auth;
-  auth.signOut().then((userCredential) => {
-    userIsAuthenticated = false;
-  })
-  .catch((error) => {
-    const errorCode = error.code;
-    const errorMessage = error.message;
-  });
-}
-
-
-
-
-
-
+  auth
+    .signOut()
+    .then((userCredential) => {
+      userIsAuthenticated = false;
+    })
+    .catch((error) => {
+      const errorCode = error.code;
+      const errorMessage = error.message;
+    });
+};
 
+// Locations Database
```

Comment:
Maybe remove commented out code if it's not needed. But if it is needed for future features, can leave it there